### PR TITLE
chore: fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Hello! 👋🏻 

I've gone through all files in the repo and fixed all typos.
Then reverted all false positives.

Please be gentle with me.

### CODE REVIEW

File: tests/library/inspector/cli-codegen-2.spec.ts [hunk 1]
```diff
@@ -462,7 +462,7 @@ await page1.GotoAsync("about:blank?foo");`);
     expect(models.active).toBe('#checkbox');
   });
 
-  test('should check input with chaning id', async ({ page, openRecorder }) => {
+  test('should check input with chaining id', async ({ page, openRecorder }) => {
```
Comment (szepeviktor) [inline tests/library/inspector/cli-codegen-2.spec.ts:465]: I am not sure here.

**Comment by szepeviktor** [comment]
@microsoft-github-policy-service agree

File: tests/library/inspector/cli-codegen-2.spec.ts (hunk 1 above)
Comment (mxschmitt) [reply tests/library/inspector/cli-codegen-2.spec.ts:465]: this one looks good

**Review by mxschmitt — COMMENTED** [review]
Please revert changes inside:

- tests/assets/wpt/
- browser_patches/

directories.

File: packages/playwright-core/src/server/chromium/crPage.ts [hunk 2]
```diff
@@ -515,7 +515,7 @@ class FrameSession {
           // Note: frames might be removed before we send these.
           this._client._sendMayFail('Page.createIsolatedWorld', {
             frameId: frame._id,
-            grantUniveralAccess: true,
```
[the remark this answers is not in the document]
Comment (szepeviktor) [reply packages/playwright-core/src/server/chromium/crPage.ts]: Done 🍏

**Comment by szepeviktor** [comment]
Reverted.

**Review by mxschmitt — APPROVED** [review]

**Comment by szepeviktor** [comment]
Glad to contribute.

**Comment by szepeviktor** [comment]
For the willing contributor: `.typos.toml`
```toml
[files]
extend-exclude = [
    ".git/",
    "tests/assets/wpt/",
    "browser_patches/",
    "tests/assets/react/",
    "tests/assets/reading-list/",
    "tests/assets/stress/static/",
    "packages/playwright-core/src/server/chromium/protocol.d.ts",
    "packages/playwright-core/src/server/firefox/protocol.d.ts",
    "packages/playwright-core/src/server/webkit/protocol.d.ts",
    "packages/playwright-core/types/protocol.d.ts",
]
ignore-hidden = false

[default]
extend-ignore-re = [
    "Lorem ipsum .*",
    "heure normale de",
    "\\bdisable-fre\\b",
    "\\borg\\.testng\\.",
    'id="[0-9A-Za-z-]{3}-[0-9A-Za-z-]{2}-[0-9A-Za-z-]{3}"',
    "\\{ name: 'name1', value: '[^']+',",
    "SUite[234]",
]

[default.extend-words]
"bitap" = "bitap"

[default.extend-identifiers]
"BigSur" = "BigSur"
"comparePNGs" = "comparePNGs"
"HPE_INVALID_EOF_STATE" = "HPE_INVALID_EOF_STATE"
"HPE_INVALID_METHOD" = "HPE_INVALID_METHOD"
"multiplebgs" = "multiplebgs"
"nsIInterfaceRequestor" = "nsIInterfaceRequestor"
# What does it mean??
"scoreBouns" = "scoreBouns"
"grantUniveralAccess" = "grantUniveralAccess"

# Typos
"Github" = "GitHub"
```

**Pull request merged**